doc/changes.md

```diff
@@ -1,5 +1,76 @@
 # Changelog
 
+## Release v0.3.1 (2021/03/14)
+
+[![Digital Object Identifier for PyGMT v0.3.1](https://zenodo.org/badge/DOI/10.5281/zenodo.3781524.svg)](https://doi.org/10.5281/zenodo.3781524)
+
+### Highlights
+
+*🎉 **Multiple bug fixes and a reorganized gallery** 🎉
```

Comment:
I think we can add one more highlight (usually we have 3 in total). Any more suggestions?

Reply:
Then I'd like to recommend another gallery example: [Vector heads and tails](https://www.pygmt.org/dev/gallery/lines/vector_heads_tails.html#sphx-glr-gallery-lines-vector-heads-tails-py). It is cool and useful.